### PR TITLE
Adds a new sodie pop and a caffeine reagent for later use

### PR DESCRIPTION
**Pull request by veles2001**
[content][sprites]
I was originally going to make a Zero Ultra meme clone but I decided against it. The drink is a premium item at Groans Soda. Sorry for the shit sprite, spriting really isn't my thing.
sprite
In a later pull request, I might change coffee to use caffeine instead of a dedicated coffee reagent because at the moment, they do roughly the same thing.
:cl:
 * rscadd: Added caffeine reagent
 * rscadd: Added Roentgen Energy
 * imageadd: Added sprite for Roentgen Energy

### CODE REVIEW

**Comment by Goldsteel** [comment]
"reagents.add_reagent(URANIUM, 3.6)"

image

It's so good your tastebuds will meltdown

**Comment by SonixApache** [comment]
\>trying to implement heart destruction for coffee drinkers
😔

**Comment by veles2001** [comment]
Then what do you suggest for caffeine overdose?

**Comment by SonixApache** [comment]
tunnelvision, blackouts, stomach aches, hunger, a combination of all those or even better; no overdose at all

**Comment by veles2001** [comment]
I can settle for hunger. It fulfills my autism criteria and isn't too extreme. I'll handle it later today because I'm going to sleep soon.

File: code/modules/reagents/reagent_containers/food/drinks.dm [hunk 1]
```diff
@@ -763,6 +763,17 @@
 	reagents.add_reagent(COCAINE, 5)
 	reagents.add_reagent(BLACKCOLOR, 5)
 
+/obj/item/weapon/reagent_containers/food/drinks/soda_cans/roentgen_energy
+	name = "Roentgen Energy"
```
Comment (Kurfursten) [inline code/modules/reagents/reagent_containers/food/drinks.dm]: \improper

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 2]
```diff
@@ -4707,6 +4707,23 @@
 	color = "#D9C0E7" //rgb: 217, 192, 231
 	custom_metabolism = 0.1
 
+/datum/reagent/caffeine
+	name = "Caffeine"
+	id = CAFFEINE
+	description = "Caffeine is a stimulant that is commonly found in coffee and tea. It works by making your metabolism faster so it also increases your appetite."
```
Comment (Kurfursten) [inline code/modules/reagents/Chemistry-Reagents.dm]: This description is confusing since it is not currently in any coffee or tea

**Comment by SonixApache** [comment]
cute

**Comment by Blithering** [comment]
We have lived to see a world where groans soda energy drink is safer than robust softdrinks brand

**Comment by SonixApache** [comment]
Blithering brings up a good point; throw this in the Groans machine instead.

**Comment by jknpj** [comment]
remember to update the wiki

**Pull request merged**